src/components/Learn.js

```diff
@@ -11,10 +11,12 @@ import { LinearGradient } from 'expo-linear-gradient'
 const { width, height } = Dimensions.get('window')
 const SHOP_IMAGE_RATIO = 599/416
 const SHOP_IMAGE_WIDTH = width * .6
-const STORE_OPTION_FONT_SIZE = height * .05
+const STORE_OPTION_FONT_SIZE = height * .03
 const SHOP_IMAGE_HEIGHT = SHOP_IMAGE_WIDTH / SHOP_IMAGE_RATIO
+const ICON_SIZE = width * .08
+const ICON_COLOR = 'white'
 
-const Store = ({ navigation }) => {
+const Learn = ({ navigation }) => {
```

Comment:
I just realized you removed the marker store entirely from the app, do you think this is because it's a bad feature? If we agree to remove it then CustomMarker.js would need to be removed, same as MarkerStyleStore and other parts of the code should be removed so there's no pollution

Reply:
Mhhhh 🤔🤔🤔🤔🤔

Reply:
Don't understand this one but think it's just a "kiproko" like we said